build-tools/makefile_components/base_container_arm64.mak

```diff
@@ -0,0 +1,28 @@
+# Container section of standard makefile (arm64)
+##### PLEASE DO NOT CHANGE THIS FILE #####
+##### If one of these sections does not meet your needs, consider copying its
+##### contents into ../Makefile and commenting out the include and adding a
+##### comment about what you did and why.
+
+SANITIZED_DOCKER_REPO = $(subst /,_,$(DOCKER_REPO))
+
+DOTFILE_IMAGE = $(subst /,_,$(IMAGE))-$(VERSION)
+
+container-arm64: .container-arm64-$(DOTFILE_IMAGE) container-arm64-name
+
+# In CI environments, use the plain Docker build progress to not overload the CI logs
+PROGRESS := $(if $(CI),plain,auto)
+
+.container-arm64-$(DOTFILE_IMAGE): $(wildcard Dockerfile Dockerfile.in) container-arm64-name
```

Comment:
The Dockerfile.in stuff is from ancient time before multi-stage builds were available, so completely irrelevant and obsolete. I understand where it's coming from, but it's probably easier to just have `Dockerfile` here instead of the wildcard statement.